sassc.py

```diff
@@ -40,7 +40,12 @@
    Watch file for changes.  Requires the second argument (output CSS
    filename).
 
+   Note that ``--watch`` does not understand imports.  Due to this, the
```

Comment:
It would be better if we use [`.. note::`](http://docutils.sourceforge.net/docs/ref/rst/directives.html#note) here.